src/loaders/ObjectLoader.js

```diff
@@ -466,8 +466,32 @@ THREE.ObjectLoader.prototype = {
 
 				case 'PerspectiveCamera':
 
-					object = Object.assign(
-							new THREE.PerspectiveCamera(), data );
+					if ( data.focalLength !== undefined ) {
```

Comment:
Also, I think we can ignore `data.focalLength`. It was used only briefly for `StereCamera` in the previous cycle. I don't think anyone got to rely on it being serialised/deserialised.